src/components/timeline/timeline.component.less

```diff
@@ -16,8 +16,8 @@ ux-timeline {
             margin-left: -1.5px;
         }
 
-        &:after:extend(.hpe-chevron-down:before) {
-            font-family: 'hpe-icons';
+        &:after:extend(.ux-icon-chevron-down:before) {
+            font-family: 'ux-icons';
```

Comment:
I don't think I understand this. Can the icon be defined within the component itself, given a class, and the styling be applied to that class?